### DISCUSSION TITLE
Interfaces Just White

### DISCUSSION THREAD

**Discussion by ghost** [whs/runekit#45] [Q&A Linux]
Any time I've tried to install RuneKit, I've run into this problem. Basically, the tray icon works great. However, all windows (e.g. Stats) just open a white box with window decorations.

Anyone know what I could do about this?

New Linux Mint 21.3 install with Cinnamon & Wine-Staging (if that matters)

Thanks for any help

**Comment by Ilwyd** [comment]
Late response, but I'm also having a similar issue on NixOS. Windows open, but they're just the border with nothing inside.

![image](https://github.com/user-attachments/assets/573479fc-f7ee-461e-83ab-40fc03c43425)

**Comment by Ilwyd** [comment]
Found a thread on Reddit which resolved the issue for me. Needed to set this env variable: `QTWEBENGINE_CHROMIUM_FLAGS="--no-sandbox"`
